ansible/controller-lean.yml

```diff
@@ -0,0 +1,39 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more contributor
```

Comment:
do we really need this file if the image name for the two version of the controller is the same?

Reply:
yes, we need it for several reasons. 
- specify that current setup is actually lean
- specify lean.yml as plugin
would be happy to get rid of it, but can't find a good way to specify those

Reply:
as discussed on slack - i think it's possible to nuke this with a task to set lean mode.

Reply:
Done. controller-lean.yml removed